components/suggestion/modal_suggestion_list.tsx

```diff
@@ -39,30 +75,31 @@ export default class ModalSuggestionList extends React.PureComponent {
         return {top: 0, bottom: 0, width: 0};
     }
 
-    onModalScroll = (e) => {
-        if (this.state.scroll !== e.target.scrollTop &&
+    onModalScroll = (e: Event) => {
+        const event = e.target as HTMLElement;
```

Comment:
you dont have to type cast it like this if you use event as 
`ChangeEvent<HTMLDivElement>`
where ChangeEvent is from React